components/LinearProgressWithLabel.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import LinearProgress from '@mui/material/LinearProgress'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import PropTypes from 'prop-types'
+
+export const LinearProgressWithLabel = (props) => {
+  const { status, progress, total, hideProgress } = props
+  return (
+    <>
+      {!hideProgress && (
```

Comment:
- [ ] LinearProgressWithLabel should always show a label and only hide the `<LinearProgress>` Box wrapper with `hideProgress`

Move hide progress to the `<LinearProgress>` component. This component would return an empty fragment which is not very useful. Hide this component from where it is being used based on the current state.

```jsx
// ./pages/send-assets
const hasStatusBar = useMemo(()=>{
    return typeof status === 'undefined' // add move conditions
}, [status])

{ hasStatusBar && <LinearProgressWithLabel
  status={status}
  progress={progress}
  total={total}
  hideProgress={hideProgress}
/>}

```